packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -305,7 +304,6 @@ function bindMeetingEvents(meeting) {
 
   meeting.on('meeting:startedSharingRemote', () => {
     // Set the source of the video element to the previously stored stream
```

Comment:
add a check for remoteShareStream then set it else dont set it , Sometimes a user is already sharing initially so tthe vent will come before the actual remote sttreams